pkg/ldd/ldd_unix.go

```diff
@@ -80,13 +79,16 @@ func runinterp(interp, file string) ([]string, error) {
 		return nil, err
 	}
 	for _, p := range strings.Split(string(o), "\n") {
-		f := strings.Split(p, " ")
+		f := strings.Split(strings.TrimSpace(p), " ")
```

Comment:
Will strings.Fields(strings.TrimSpace(p)) work here (https://pkg.go.dev/strings#Fields)?  That will split by "one or more consecutive white space characters"

Reply:
Oh, that is neat. Never seen that function before.